src/DotNetLightning.Core/Channel/ChannelValidation.fs

```diff
@@ -90,45 +92,45 @@ module internal ChannelHelpers =
+                                          remoteChannelKeys.PaymentBasePubKey
                                           localParams.IsFunder
                                           localParams.DustLimitSatoshis
                                           revPubKeyForLocal
                                           remoteParams.ToSelfDelay
                                           delayedPubKeyForLocal
                                           paymentPubKeyForLocal
-                                          (localParams.ChannelPubKeys.HTLCBasePubKey)
-                                          (remoteParams.HTLCBasePoint)
+                                          (localChannelKeys.HTLCBasePubKey)
+                                          (remoteChannelKeys.HTLCBasePubKey)
                                           localSpec
                                           n
-            let revPubKeyForRemote = Generators.revocationPubKey secpContext localParams.ChannelPubKeys.RevocationBasePubKey remotePerCommitmentPoint.PubKey
-            let delayedPubKeyForRemote = Generators.derivePubKey secpContext remoteParams.DelayedPaymentBasePoint remotePerCommitmentPoint.PubKey
-            let paymentPubKeyForRemote = Generators.derivePubKey secpContext localParams.ChannelPubKeys.PaymentBasePubKey remotePerCommitmentPoint.PubKey
+            let revPubKeyForRemote = Generators.revocationPubKey secpContext localChannelKeys.RevocationBasePubKey remotePerCommitmentPoint.PubKey
+            let delayedPubKeyForRemote = Generators.derivePubKey secpContext remoteChannelKeys.DelayedPaymentBasePubKey remotePerCommitmentPoint.PubKey
+            let paymentPubKeyForRemote = Generators.derivePubKey secpContext localChannelKeys.PaymentBasePubKey remotePerCommitmentPoint.PubKey
             let remoteCommitTx =
                 Transactions.makeCommitTx scriptCoin
                                           CommitmentNumber.FirstCommitment
-                                          remoteParams.PaymentBasePoint
-                                          localParams.ChannelPubKeys.PaymentBasePubKey
+                                          remoteChannelKeys.PaymentBasePubKey
+                                          localChannelKeys.PaymentBasePubKey
                                           (not localParams.IsFunder)
                                           (remoteParams.DustLimitSatoshis)
                                           revPubKeyForRemote
                                           localParams.ToSelfDelay
                                           delayedPubKeyForRemote
                                           paymentPubKeyForRemote
-                                          (remoteParams.HTLCBasePoint)
-                                          (localParams.ChannelPubKeys.HTLCBasePubKey)
+                                          (remoteChannelKeys.HTLCBasePubKey)
+                                          (localChannelKeys.HTLCBasePubKey)
```

Comment:
nit: let's remove the parenthesis now that you're changing these lines, they are not needed

Reply:
This, and the other nit below, get fixed in a later commit anyway when these lines change again.